task03/Calculate.java

```diff
@@ -0,0 +1,62 @@
+//This program does 4 math operations (+, -, *, /). Each operation has there own method.
+public class Calculate {
+    //Method returns sum of parameters
+    private static double sum(double param1, double param2) {
+        return param1 + param2;
+    }
+
+    //Method returns difference of parameters
+    private static double difference(double param1, double param2) {
+        return param1 - param2;
+    }
+
+    //Method multiplies parameters and return result
+    private static double multiply(double param1, double param2) {
+        return param1 * param2;
+    }
+
+    //Method verifies parameters and return quotient
+    private static double quotient(double param1, double param2) {
+        if (param2 == 0) {
```

Comment:
It's not correct to compare two doubles in such a way. Please take a look to http://www.cygnus-software.com/papers/comparingfloats/Comparing%20floating%20point%20numbers.htm
In this case it's even better to perform division but check result for special values